admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -610,6 +610,13 @@ var specificPrices = (function() {
     });
   }
 
+  function resetForm() {
+    /**
```

Comment:
This is intended. From my own experience, multi lines comment are more visible. This comment is *VERY* important, regarding the impact on the back office.